@navikt/core/react/src/table/ExpandableRow.tsx

```diff
@@ -110,7 +111,7 @@ export const ExpandableRow: ExpandableRowType = forwardRef(
             className={cl("navds-table__toggle-expand-cell", {
               "navds-table__toggle-expand-cell--open": _open,
             })}
-            onClick={expansionHandler}
+            onClick={!expansionDisabled ? expansionHandler : () => null}
```

Comment:
I find this easier to read:
```suggestion
            onClick={expansionDisabled ? undefined : expansionHandler}
```